packages/shared/world/parcelSceneManager.ts

```diff
@@ -186,6 +186,63 @@ export const parcelSceneLoadingState: ParcelSceneLoadingState = {
   isolatedModeOptions: null as IsolatedModeOptions | null
 }
 
+async function isEmptyParcel(coord: string): Promise<boolean> {
+  if (!parcelSceneLoadingState.lifecycleManager) return false
+
+  for (const record of parcelSceneLoadingState.lifecycleManager.sceneIdToRequest) {
```

Comment:
Is it possible to generate the empty scene id and just check if `parcelSceneLoadingState.lifecycleManager.sceneIdToRequest.get(emptySceneId)` is undefined?

Reply:
No, I don't remember exactly why but I remember that we needed to check the base position of the scene because it could change of size when we invalidate it